lntest/itest/lnd_test.go

```diff
@@ -4572,32 +4567,56 @@ func assertReports(ctxb context.Context, t *harnessTest,
 }
 
 // findSweep looks up a sweep in a nodes list of broadcast sweeps.
-func findSweep(ctx context.Context, node *lntest.HarnessNode,
-	sweep *chainhash.Hash) error {
+func findSweep(ctx context.Context, t *testing.T, node *lntest.HarnessNode,
```

Comment:
nit: now that it's asserting, should we rename to `assertSweepFound` or something similar?